src/canorus.cpp

```diff
@@ -87,61 +88,66 @@ void CACanorus::initMain( int argc, char *argv[] ) {
 	Initializes language specific settings like the translation file for the GUI,
 	text flow (left-to-right or right-to-left), default string encoding etc.
  */
-void CACanorus::initTranslations() {
+void CACanorus::initTranslations(/*QTranslator &translator*/) {
 	QString translationFile = "lang:" + QLocale::system().name() + ".qm"; // load language_COUNTRY.qm
 	if(!QFileInfo(translationFile).exists())
 		translationFile = "lang:" + QLocale::system().name().left(2) + ".qm"; // if not found, load language.qm
 
-	QTranslator *translator = new QTranslator(); // translators are destroyed when application closes anyway
+    std::unique_ptr<QTranslator> translator(new QTranslator);
```

Comment:
Is `std::unique_ptr` the right thing to use? Should we use something more Qt-specific?